templates/internet-of-things/management.html

```diff
@@ -88,126 +110,170 @@ <h3>Integrate with cloud solutions</h3>
-                            alt="",
-                            width="32",
-                            height="28",
-                            hi_def=True,
-                            loading="lazy",
-                            attrs={"class": "p-heading-icon__img p-heading-icon__img--small"},) | safe
-              }}
-              <h3 class="p-heading-icon__title p-heading--4">Whitepaper</h3>
-            </div>
-          </div>
+
+    <section class="p-section">
+      <div class="row--50-50 p-section--shallow">
+        <hr />
+        <div class="col">
+          <h2>Enterprise-grade solutions for every scenario</h2>
+        </div>
+        <div class="col">
           <p>
-            <a class="p-link--inverted"
-               href="/engage/iot-device-management-whitepaper?_ga=2.90790234.1537229770.1638375570-1154790289.1626446765">Secure IoT device management&nbsp;&rsaquo;</a>
+            We know that everyone's needs are different. That's why we have multiple solutions available for you to use to manage your devices.
           </p>
-        </div>
-        <div class="col-4 p-divider__block">
-          <div class="p-heading-icon">
-            <div class="p-heading-icon__header">
-              {{ image(url="https://assets.ubuntu.com/v1/816ae23b-Webinar+-+white.svg",
-                            alt="",
-                            width="32",
-                            height="28",
-                            hi_def=True,
-                            loading="lazy",
-                            attrs={"class": "p-heading-icon__img p-heading-icon__img--small"},) | safe
-              }}
-              <h3 class="p-heading-icon__title p-heading--4">Webinar</h3>
-            </div>
-          </div>
+          <hr class="p-rule--muted" />
           <p>
-            <a class="p-link--inverted"
-               href="/engage/snap-store-proxy-intro-webinar">A technical introduction to the Snap Store Proxy&nbsp;&rsaquo;</a>
+            <a href="/internet-of-things/contact-us?product=management">Get in touch to discuss the best solution for you&nbsp;&rsaquo;</a>
           </p>
         </div>
-        <div class="col-4 p-divider__block">
-          <div class="p-heading-icon">
-            <div class="p-heading-icon__header">
-              {{ image(url="https://assets.ubuntu.com/v1/4ab8ff35-Whitepaper+-+white.svg",
-                            alt="",
-                            width="32",
-                            height="28",
-                            hi_def=True,
-                            loading="lazy",
-                            attrs={"class": "p-heading-icon__img p-heading-icon__img--small"},) | safe
-              }}
-              <h3 class="p-heading-icon__title p-heading--4">Whitepaper</h3>
+      </div>
+      <div class="row--25-75-on-large">
+        <div class="col">
+          <div class="p-equal-height-row has-divider-1">
```

Comment:
The images in these cols look very large compared to the corresponding section on https://ubuntu.com/desktop which we're basing off of, could we make them smaller?

Reply:
@mattea-turic what's your call on this?

Reply:
I have reduced the size of images, let me know if that works.